src/main/java/com/wvu/ai/csp/project/SudokuSolver/App.java

```diff
@@ -8,22 +8,34 @@
  * Created by Shane Hogan on 4/9/16.
  */
 public class App {
-    //TODO replace size with user input
-    private static int SIZE = 9;
+    private static int SIZE;
     static  Scanner kbd = new Scanner(System.in);
```

Comment:
This should be marked private, the name should shifted to all caps, and at the end of the main method, the Scanner should be closed.